SPECS/shim-unsigned-x64/shim-unsigned-x64.spec

```diff
@@ -1,80 +1,284 @@
+%global pesign_vre 0.106-1
+%global openssl_vre 1.0.2j
+
+# For prereleases, % global prerelease rc2, and downpatch Makefile
+%if %{defined prerelease}
+%global dashpre -%{prerelease}
+%global dotpre .%{prerelease}
+%global tildepre ~%{prerelease}
+%global zdpd 0%{dotpre}.
+%endif
+
+%bcond altarch 0
+
+%global efidir azurelinux
+%global shimrootdir %{_datadir}/shim/
+%global shimversiondir %{shimrootdir}/%{version}
+%global efiarch x64
+%global shimdir %{shimversiondir}/%{efiarch}
+%if %{with altarch}
+%global efialtarch ia32
+%global shimaltdir %{shimversiondir}/%{efialtarch}
+%endif
+
 %global debug_package %{nil}
-Summary:        First stage UEFI bootloader
-Name:           shim-unsigned-x64
-Version:        15.4
-Release:        3%{?dist}
-License:        BSD
-Vendor:         Microsoft Corporation
-Distribution:   Azure Linux
-URL:            https://github.com/rhboot/shim
-Source0:        https://github.com/rhboot/shim/releases/download/%{version}/shim-%{version}.tar.bz2
-Source1:        sbat.csv.in
-Source100:      cbl-mariner-ca-20211013.der
-Patch0:         Don-t-call-QueryVariableInfo-on-EFI-1.10-machines.patch
-Patch1:         Fix-handling-of-ignore_db-and-user_insecure_mode.patch
-Patch2:         Fix-a-broken-file-header-on-ia32.patch
-Patch3:         mok-allocate-MOK-config-table-as-BootServicesData.patch
-Patch4:         shim-another-attempt-to-fix-load-options-handling.patch
-Patch5:         Relax-the-check-for-import_mok_state.patch
-BuildRequires:  dos2unix
-BuildRequires:  vim-extra
-ExclusiveArch:  x86_64
+%global __debug_package 1
+%global _binaries_in_noarch_packages_terminate_build 0
+%if %{with altarch}
+%global __debug_install_post %{SOURCE100} %{efiarch} %{efialtarch}
+%else
+%global __debug_install_post %{SOURCE100} %{efiarch}
+%endif
+%undefine _debuginfo_subpackages
+
+# currently here's what's in our dbx: nothing
+%global dbxfile %{nil}
+
+Name:		shim-unsigned-%{efiarch}
+Version:	15.8
+Release:	3%{?dist}
+Summary:	First-stage UEFI bootloader
+ExclusiveArch:	x86_64
+License:	BSD
+Vendor:		Microsoft Corporation
+Distribution:	Azure Linux
+URL:		https://github.com/rhboot/shim
+Source0:	https://github.com/rhboot/shim/releases/download/%{version}%{?dashpre}/shim-%{version}%{?dotpre}.tar.bz2
+Source1:	azurelinux-ca-20230216.der
+%if 0%{?dbxfile}
+Source2:	%{dbxfile}
+%endif
+Source3:	sbat.azurelinux.csv
+Source4:	shim.patches
```

Comment:
Renders empty to me. Probably want to list the patch files. I know the PR is not ready for review. Just pre-emptively iterating of it.

Reply:
looks to be an artifact from using the fedora spec :) https://github.com/microsoft/azurelinux/pull/10995/commits/73c0c0065cac90bca75ea7a5fc30981e1f917651